cinema/urls.py

```diff
@@ -1,10 +1,26 @@
-from django.urls import path
+from django.urls import path, include
 
-from cinema.views import movie_list, movie_detail
+from rest_framework import routers
+from cinema.views import (
+    GenreList,
+    GenreDetail,
+    CinemaHallViewSet,
+    ActorList,
+    ActorDetail,
+    MovieViewSet
+)
+
+
+router = routers.DefaultRouter()
+router.register("movies", MovieViewSet)
+router.register("cinema_halls", CinemaHallViewSet)
 
 urlpatterns = [
-    path("movies/", movie_list, name="movie-list"),
-    path("movies/<int:pk>/", movie_detail, name="movie-detail"),
+    path("", include(router.urls)),
```

Comment:
No need include if basepath not provided
You can do:
urlpatterns = [list of urls ]+ router.urls